closed/src/java.base/share/classes/openj9/internal/criu/NativePRNG.java

```diff
@@ -239,11 +273,100 @@ public OutputStream run() {
                     try {
                         seedOut.write(seed);
                     } catch (IOException e) {
-                        // Ignored. On Mac OS X, /dev/urandom can be opened
-                        // for write, but actual write is not permitted.
+                        // ignored, On Mac OS X, /dev/urandom can be opened
```

Comment:
On -> on (no capital in the middle of a phrase).

Reply:
updated